Unreal Engine plugin documentation update (#6350)
getsentry/sentry-docs: src/platform-includes/getting-started-primer/unreal.mdx

- Native support for automatic crash error tracking for
  - Android by using the [Android SDK](/platforms/android/) to support Java, Kotlin, C and C++
  - iOS by using the [iOS SDK](/platforms/apple/guides/ios/) to support Objective-C, Swift, C and C++
  - Windows and Linux by using the [Native SDK](/platforms/native/) to support C and C++ with minidumps
  - macOS by using the [macOS SDK](/platforms/apple/guides/macos/) to support Objective-C, Swift, C and C++
- Compatible with Unreal Engine 5
- [Crash Reporter Client](/platforms/unreal/setup-crashreporter/) provided along with Unreal Engine

<Alert level="warning">

> In order to capture crashes on Windows automatically [Crash Reporter Client](/platforms/unreal/setup-crashreporter/) has to be configured. Starting from UE 5.2 this step is not required since the engine provides an API allowing to switch between the default and third-party crash handling solutions.

Reviewer: ```suggestion
[Crash Reporter Client](/platforms/unreal/setup-crashreporter/) has to be configured in order to capture crashes on Windows automatically. This step isn't required starting with UE 5.2 because the engine provides an API, which allows you to switch between default and third-party crash-handling solutions.
```
